### ISSUE TITLE
levitation problem

### ISSUE THREAD

**Issue by ButterWalled** [sakisiil/Origin-Datapacks#4]
since i have the mob origins installed, this doesnt want to work. should i move it out of the mod map in order for it to work?

**Comment by Phenicwashere** [comment]
@ButterWalled  i don't think it has anything to do with the mob origins, i have the mob origins and the bee origin and everything works just fine
